ding2.profile

```diff
@@ -829,3 +829,24 @@ function ding2_set_cookie_page() {
   // display EU Cookie Compliance popup: anonymous user, authenticated user
   // administer EU Cookie Compliance popup: administrators, local administrator
 }
+
+/**
+ * Enabling Shortcuts plugin for Administration Menu module.
+ */
+function ding2_admin_menu_shortcuts() {
+  if (module_exists('admin_menu')) {
+    $content = variable_get('admin_menu_components', array());
+
+    if (empty($content)) {
+      module_load_include('inc', 'admin_menu', 'admin_menu');
+      admin_menu_theme_settings();
```

Comment:
It's truly hard to recall what you've been doing 6 months ago.
We don't assign this function to any variable and its not used. I'll remove it.